x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/_legend.scss

```diff
@@ -6,7 +6,7 @@
   height: $euiSizeM;
   width: $euiSizeM;
   background-color: $euiColorGhost;
-  border: 1px solid $euiColorVis2;
+  border: 2px solid $euiColorVis2;
```

Comment:
You could consider using an `$euiBorderWidthThick` for these `2px` values.

Reply:
Updated in 626ab62bb02d68546b25cf3cbf527a0fd88cc1a9